qa/acceptance/spec/config_helper.rb

```diff
@@ -0,0 +1,19 @@
+require "json"
```

Comment:
we, no problem! I stick too much to old behaviour if using plain json parser, 👍 I should remember myself all time!